packages/components/src/components.d.ts

```diff
@@ -232,7 +232,7 @@ export namespace Components {
     */
     'disableInteractivity': boolean;
     /**
-    * Control the text shown in header subtitle
+    * Use this widget to put the widget in "error mode". When error mode is active. The header will display the given text. And the body will be display the errorDescription instead any data.
```

Comment:
```suggestion
    * Use this widget to put the widget in "error mode". When error mode is active the header will display the given text and the body will display the `errorDescription` instead of the widget content.
```